include/circt/Dialect/FIRRTL/AnnotationDetails.h

```diff
@@ -48,10 +48,6 @@ constexpr const char *traceNameAnnoClass =
 constexpr const char *traceAnnoClass =
     "chisel3.experimental.Trace$TraceAnnotation";
 
-constexpr const char *omirAnnoClass =
-    "freechips.rocketchip.objectmodel.OMIRAnnotation";
-constexpr const char *omirFileAnnoClass =
-    "freechips.rocketchip.objectmodel.OMIRFileAnnotation";
 constexpr const char *omirTrackerAnnoClass =
     "freechips.rocketchip.objectmodel.OMIRTracker";
```

Comment:
This can also go. If you don't handle it in this PR, I can deal with with it in a follow-on. 

This does mean that the `shouldBeDeleted` (or can be) function needs to be updated.

Reply:
Yep, I'll pull this thread

Reply:
This is now going deeper than I wanted... I'm starting to look at the changes to IMDCE and I'm nervous I've missed something... So, I broke it out and I've sent a draft PR here, based on this branch for now: https://github.com/llvm/circt/pull/7908